ansible-scylla-node/tasks/common.yml

```diff
@@ -277,3 +277,20 @@
   become: true
   when: update_sysctl|bool is defined and update_sysctl|bool
 
+- name: Deactivate firewall
+  block:
+    - name: Deactivate front-end firewall services
+      service:
+        name: "{{ item }}"
+        state: stopped
```

Comment:
what will happen if the machine will be restarted?
will the firewalls start back up?
what if ufw is not installed? (you have ignore errors, but I don't think this is a good solution, if you ignore error, then later you will figure out the firewall is not down and your connection doesn't work)